crates/orchestrator/src/tests/jobs/mod.rs

```diff
@@ -101,7 +104,7 @@ async fn create_job_job_exists_in_db_works() {
 
     // Queue checks.
     let consumed_messages =
-        services.config.queue().consume_message_from_queue(JOB_PROCESSING_QUEUE.to_string()).await.unwrap_err();
+        services.config.queue().consume_message_from_queue(PROVING_JOB_PROCESSING_QUEUE.to_string()).await.unwrap_err();
     assert_matches!(consumed_messages, QueueError::NoData);
```

Comment:
The trait QueueName makes sense here since now the code is highly dependent on the developer
We could create a variable with job type:
```RUST
let job_type = JobType::ProofCreation;
build_job_item_by_type_and_status(job_type, JobStatus::Created, "0".to_string());
// rest code
services.config.queue().consume_message_from_queue(job_type.processing_queue_name()).await.unwrap_err();
```
This is much less depended on the developer now and creates a single point of failure.

This needs to be changed in all the test cases you are using any type of queue.

Reply:
updated